iosApp/iosApp/Pages/StopDetails/DirectionPicker.swift

```diff
@@ -32,7 +32,7 @@ struct DirectionPicker: View {
     var body: some View {
         if availableDirections.count > 1 {
             let deselectedBackroundColor = deselectedBackgroundColor(route)
-            HStack(alignment: .center) {
+            HStack(alignment: .center, spacing: 2) {
```

Comment:
This doesn't actually matter, but the default of 8 was visibly different from the Android status quo of 0, so I checked what Figma technically calls for so that we can use it on both iOS and Android.